aws/rust-runtime/aws-types/src/sdk_config.rs

```diff
@@ -602,4 +602,22 @@ impl SdkConfig {
     pub fn builder() -> Builder {
         Builder::default()
     }
+
+    /// Convert this [`SdkConfig`] back to a builder to enable modification
+    pub fn to_builder(self) -> Builder {
```

Comment:
This is awesome!

The `to_builder()` on the generated configs takes `&self` rather than `self`. Should this be consistent?